locale/en/docs/guides/anatomy-of-an-http-transaction.md

```diff
@@ -126,7 +126,7 @@ request.on('error', function(err) {
 });
 ```
 
-There are other ways of [handling these errors][] such as using [`domains`][] or
+There are other ways of [handling these errors][] such as
```

Comment:
With the specific example of `domains` removed, the phrase `such as other abstractions and tools` seems a little awkward and uninformative. I'd remove those words too. I'd even consider removing the entire sentence.